lib/scoped/find.js

```diff
@@ -18,8 +23,22 @@ function find (type, api, objectsOrIds) {
       if (storedObjects.type === type) {
         return storedObjects
       } else {
-        throw new Error('Item not found')
+        throw handleIncorrectTypeError(type, storedObjects)
       }
     }
   })
 }
+
+function handleIncorrectTypeError (type, storedObject) {
+  var errorMessage = 'Object with type "' + type + '" and id "' + storedObject.id + '" is missing'
+  var incorrectType = new Error(errorMessage)
+  incorrectType.name = 'Not found'
+  incorrectType.status = 404
+  return incorrectType
+}
+
+function allObjectsAreNotFound (typedObjects) {
+  return typedObjects.every(function (typedObject) {
```

Comment:
Is the use of `Array.every` ok here?  MDN shows the following compatibility tables:
![screenshot from 2016-08-27 00-52-12](https://cloud.githubusercontent.com/assets/75311/18022605/8b74c3fc-6bf0-11e6-9054-1b5e9fcab465.png)
![screenshot from 2016-08-27 00-52-53](https://cloud.githubusercontent.com/assets/75311/18022618/a0b6f762-6bf0-11e6-88f1-261d67e9d5e7.png)